### PR TITLE
something something dark side

### PR DESCRIPTION
**Pull request by esavier**
thingies

### CODE REVIEW

File: crates/schema-registry/migrations/20210215170655_init-db.sql [hunk 1]
```diff
@@ -1,4 +1,13 @@
-CREATE TYPE schema_type_enum AS ENUM ('document_storage', 'timeseries');
+-- yes we are dropping all
+-- if you want your trash untouched write your own migration
```
Comment (smores56) [inline crates/schema-registry/migrations/20210215170655_init-db.sql:2]: pls

**Review by smores56 — CHANGES_REQUESTED** [review]

File: crates/schema-registry/migrations/20210215170655_init-db.sql (hunk 1 above)
Comment (esavier) [reply crates/schema-registry/migrations/20210215170655_init-db.sql:2]: please remove or write more? :3

**Review by smores56 — APPROVED** [review]

File: crates/schema-registry/src/main.rs [hunk 2]
```diff
@@ -18,6 +19,7 @@ pub async fn main() -> anyhow::Result<()> {
     let config = Config::from_args();
 
     status_endpoints::serve();
+    sleep(Duration::from_millis(500)).await;
```
Comment (frondeus) [inline crates/schema-registry/src/main.rs:22]: Why this sleep is necessary?

Comment (esavier) [reply crates/schema-registry/src/main.rs:22]: it prevents the dark magic of mordor from finding Frodo

**Pull request merged**